Scripts/script-AssignAnalystToIncident.yml

```diff
@@ -4,28 +4,30 @@ commonfields:
 name: AssignAnalystToIncident
 system: true
 script: |
-  var userToAssign = null;
+  var userToAssign = args.username;
```

Comment:
So if the setOwner does not find the user it returns an error, so I just use the isError to verify that and provides an appropriate message:

![image](https://user-images.githubusercontent.com/5429033/26951649-9fba4ecc-4caa-11e7-9037-9aeb0dd29fb5.png)